src/rules/at-rule-blockless-newline-after/__tests__/index.js

```diff
@@ -0,0 +1,27 @@
+import {
+  ruleTester,
+} from "../../../testUtils"
+import rule, { ruleName, messages } from ".."
+
+const testRule = ruleTester(rule, ruleName)
+
+testRule("always", tr => {
+  tr.ok("@import url(\"x.css\");\n")
+  tr.ok("@charset \"UTF-8\";\n")
+  tr.ok("@namespace url(XML-namespace-URL);\n")
+  tr.ok("@import url(\"x.css\"); /* comment */\n")
+  tr.ok("@import url(\"x.css\");\r\n", "windows")
+  tr.ok("@import url(\"x.css\"); /* comment */\r\n", "windows")
+  tr.ok("@import 'x.css';", "single blockless rule")
+
```

Comment:
Can we get a nested non-standard `@rule` test too, e.g.:

```css
a {
  @extend .b;
  @extend .c;
}
```

And a corresponding flagging test.